lib/io/fileutils/fileutils.go

```diff
@@ -248,7 +248,7 @@ func RemoveFirstLine(filePath string) error {
 	defer file.Close()
 
 	// Create a temporary file to write the updated content
-	tempFile, err := os.CreateTemp("", "temp")
+	tempFile, err := os.CreateTemp("/hab/tmp", "temp")
```

Comment:
Yes, totally! I forgot to introduce a constant in here. Now I noticed that after the most recent changes, `/hab/tmp` is actually used in more than 1 place in the `fileutils` package.

The `HAB_TMP_DIR` constant is unreachable from `fileutils`, since it's declared in `components/automate-cli/cmd/chef-automate/constants.go` and importing that package into the `fileutils` package creates an import cycle.

But I've created the same constant in the `fileutils` package now. Feel free to take another look please.